baselines/powerplatform.md

```diff
@@ -222,45 +230,52 @@ All connectors except those listed below SHOULD be added to the Blocked category
 
 ### Implementation
 
-1.  Sign in to the [Power Platform admin
-    center](https://admin.powerplatform.microsoft.com/) (for GCC
-    environments sign in to the [GCC Power Platform Admin
-    Center](https://gcc.admin.powerplatform.microsoft.us/)).
+#### MS.POWERPLATFORM.2.1v1 instructions:
+1.  Sign in to your tenant environment's respective [Power Platform admin
+    center](https://learn.microsoft.com/en-us/power-platform/admin/powerapps-us-government#power-apps-us-government-service-urls).
 
-2.  On the left pane, select **Policies** then **Data Policies.**
+2.  On the left pane, select **Policies** \> **Data Policies.**
 
 3.  Select the **+ New Policy** icon to create a new policy.
 
 4.  Give the policy a suitable agency name and click **Next.**
 
-5. At the **Prebuilt connectors** section, select the connectors that
-    fit the agency’s needs.
+5.  At the **Prebuilt connectors** section, select the connectors that
+fit the agency’s needs.
 
 6.  Select a connector and click **Move to Business.**
 
 7.  If necessary (and possible) for the connector, click **Configure
 connector** at the top of the screen to change connector
 permissions.
 
-8.  Refer to Table 1 for those connectors to move which **Business/Non-Business** Category.
+8.  Refer to Table 1 for those connectors to move which **Business/Non-Business** category.
 
-9. For the default environment, move all connectors that cannot be
+9.  For the default environment, move all connectors that cannot be
 blocked to the **Blocked** category.
 
-10. At the bottom of the screen, select **Next** to move on.
+10.  At the bottom of the screen, select **Next** to move on.
 
-11. Add a customer connector pattern that fit the agency’s needs. Click
+11.  Add a customer connector pattern that fit the agency’s needs. Click
 **Next**.
 
-12. Define the scope of the policy. For the default environment select
-**Add multiple environments** and add the default environment.
+12.  Define the scope of the policy. To use this policy for all environments including
+ones created in the future select **Add all environments**.
+
+13.  Otherwise for the default environment, select **Add multiple environments** and add the default environment.
 
-13. Select the environments over which to add the policy and click **Add
-    to policy** at the top.
+14.  Select the environments over which to add the policy and click **Add
+to policy** at the top.
 
-14. Select **Next**-\> **Create Policy** to finish.
+15.  Select **Next**-\> **Create Policy** to finish.
 
-## 3. Tenant Isolation
+#### MS.POWERPLATFORM.2.2v1 instructions:
+1.  Repeat the steps above but for step **13** select the non-default environment you wish to enforce a DLP policy upon.
+
+#### MS.POWERPLATFORM.2.3v1 instructions:
+1.  Refer to steps **8** and **9** in the **MS.POWERPLATFORM.2.3v1** instructions to meet this policy.
```

Comment:
This step within MS.POWERPLATFORM.2.3v1 refers to itself for nonexistent steps 8&9 - a typo? seems reference should be to "2.1v1" steps 8&9.